t/ui/27-plugin_obs_rsync_obs_status.t

```diff
@@ -84,22 +94,12 @@ my $host = "http://127.0.0.1:$port";
 sub fake_api_server {
     my $mock = Mojolicious->new;
     $mock->mode('test');
-    $mock->routes->get(
-        '/public/build/Proj1/_result' => sub {
-            my $c = shift;
-            return $c->render(status => 200, text => $response_dirty);
-        });
-    $mock->routes->get(
-        '/public/build/Proj2/_result' => sub {
-            my $c = shift;
-            return $c->render(status => 200, text => $response_publishing);
-        });
-    $mock->routes->get(
-        '/public/build/Proj3/_result' => sub {
-            my $c = shift;
-            return $c->render(status => 200, text => $response_published);
-        });
-
+    for my $project (keys %fake_response_by_project) {
```

Comment:
I always recommend to use `sort keys` to make results reproducable.

Reply:
But the order in which these functions are called should not matter at all.

Reply:
That might be true, but if something goes wrong, results are different for different test runs. That's simply confusing and unnecessary.
I want a reproducible output, and a `sort` does not hurt here.

Reply:
I think using `sort` here is reasonably defensive, since route order is indeed significant (they match in FIFO order).